commands/kubernetes.go

```diff
@@ -26,14 +26,15 @@ import (
 	"time"
 
 	"github.com/blang/semver"
-	"github.com/digitalocean/doctl"
-	"github.com/digitalocean/doctl/commands/displayers"
-	"github.com/digitalocean/doctl/do"
 	"github.com/digitalocean/godo"
 	"github.com/google/uuid"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
+	"github.com/digitalocean/doctl"
```

Comment:
These shouldn't be moved to a separate block I believe.

Reply:
I think Goland moved these will adjust